app/models/book.rb

```diff
@@ -1,3 +1,19 @@
 class Book < ApplicationRecord
-  
+  has_many :reviews
+
+  def average_rating
+    reviews.average(:rating)
+  end
+
+  def sort_reviews_by_rating
+    reviews.order(:rating)
+  end
+
+  def highest_review
+    sort_reviews_by_rating.last
+  end
+
+  def lowest_review
+    sort_reviews_by_rating.first
+  end
```

Comment:
I love that you built these methods here rather than calling sort_reviews_by_rating.last in your views.